crawler/crawlutils.py

```diff
@@ -508,6 +509,35 @@ def snapshot(
                     overwrite=overwrite
                 )
 
+	    try:
+                container_event = eventQ.get(block=True, timeout = wait_period)
```

Comment:
i don't get this part. So this is polling for container events every 5 seconds, and if there is an event for a new contianer, then we go and start crawling it? shouldn't that happen as soon as the event occurs?